proxy/logging/LogObject.cc

```diff
@@ -1241,6 +1241,22 @@ LogObjectManager::roll_files(long time_now)
   return num_rolled;
 }
 
+void
+LogObjectManager::reopen_moved_log_files()
+{
+  for (auto &_object : this->_objects) {
+    _object->m_logFile->reopen_if_moved();
+  }
+
+  ACQUIRE_API_MUTEX("A LogObjectManager::reopen_moved_log_files");
```

Comment:
What do the A and the R at the beginning of the strings mean?

Reply:
acquire release?

Reply:
I believe @randall is correct. I copied this convention from the other examples of this in this file (such as LogObjectManager::display).